lib/packwerk/package_set.rb

```diff
@@ -27,9 +27,11 @@ def load_all_from(root_path, package_pathspec: nil)
 
       def package_paths(root_path, package_pathspec)
         bundle_path_match = Bundler.bundle_path.join("**").to_s
-
-        Dir.glob(File.join(root_path, package_pathspec, PACKAGE_CONFIG_FILENAME))
-          .map { |path| Pathname.new(path) }.reject { |path| path.realpath.fnmatch(bundle_path_match) }
+        glob_patterns = Array(package_pathspec).map do |pathspec|
```

Comment:
`Array(x)` is more flexible, as it can handle a String path e.g. `"**/"` along with an array of paths coming from a `package.yml` file
```yml
package_paths:
- components/sales
- "."
```